users/urls.py

```diff
@@ -1,10 +1,14 @@
 from django.conf.urls import url
 
-from .views import UserLoginView, DashboardView, UserLogoutView, RegisterView
+from .views import (UserLoginView, DashboardView, UserLogoutView, RegisterView,
```

Comment:
don't do this. it's prone to merge conflict

do this instead
from .views import (
    UserLoginView,
    DashboardView,
    UserLogoutView,
    ...
)